### PR TITLE
Rails-react' project as one app with Webpack

### PR DESCRIPTION
**Pull request by kessio**
Hello 👋🏾 
The following has been implemented on this PR;

1. Create a new Rails app called 'hello-rails-react'.
2. Use the command rails new -j webpack my_new_rails_app as described in the previous lesson's tutorial.
3. Verify that the jsbundling-rails gem is added to your Gemfile.
4. Initialize your project with Git.
5. Install needed node dependencies as described in the previous lesson's tutorial.
6. Create a babel.config.js in the root folder of your project as described in the previous lesson's tutorial.
7. Update webpack.config.js to include the babel loader as described in the previous lesson's tutorial.
8. Make sure that your project has a Postgres database set up.
9. Create a table for storing your messages and create 5 different greetings.
10. Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and Rails route).
11. Create a static view that will be the root of your app.
12. Create the App component with react-router and render it in your static view.
13. Create the Greeting component that will display a greeting. Set it up as a route in your App component.
14. Create a store, an action, and a reducer that will connect to your API endpoint to get the random greeting.
15. Display the random greeting in your Greeting component.

### CODE REVIEW

**Review by Davidosky007 — COMMENTED** [review]
## Sorry this was a mistake

File: db/seeds.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
```
Comment (Davidosky007) [inline db/seeds.rb:2]: * Kindly Create 5 different greetings as it's a [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/rails_react_webpack.md#:~:text=Create%20a%20table%20for%20storing%20your%20messages%20and%20create%205%20different%20greetings.)

File: README.md [hunk 2]
```diff
@@ -1 +1,212 @@
...
+  cd my-folder
+git clone https://github.com/kessio/hello-world.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd my-project
```
Comment (Davidosky007) [inline README.md:104]: * Kindly update your setup instructions as I can see that your API works after creating some greetings but starting the app using `./bin/dev` doesn't work and running also following your readme setup instructions don't work either

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @kessio 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* No linter errors 👍 
* Well written PR description 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1 +1,211 @@
...
+ 
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+
+```sh
+  rails server
```
Comment (prolajumokeoni) [inline README.md:118]: - [ ] Kindly update the instructions for starting the application locally `./bin/dev`

**Review by prolajumokeoni — APPROVED** [review]
Hi @kessio 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- App setup as one app with webpack 👏


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**